Knossos.NET/ViewModels/Templates/DevModMembersMgrViewModel.cs

```diff
@@ -104,6 +104,13 @@ public async void UpdateUI()
             if (editor != null && !MemberItems.Any())
             {
                 if (Nebula.userIsLoggedIn) {
+                    //verify if the modid is already uploaded to nebula
+                    if(await Nebula.IsModIdInNebula(editor.ActiveVersion.id) == false)
+                    {
+                        _ = MessageBox.Show(MainWindow.instance!, "The mod id: " + editor.ActiveVersion.id + ", was not found in the nebula database. This likely mean you mod was never uploaded to nebula.\nYou need to upload at least one version to nebula (public or private) in order to manage mod members.\nIt can also be caused by a network error." , "Mod ID not in Nebula", MessageBox.MessageBoxButtons.OK);
```

Comment:
Here's a version that fixes a grammar issue in the second sentence and ensures consistent capitalization of 'Nebula'

_ = MessageBox.Show(MainWindow.instance!, "The mod id: " + editor.ActiveVersion.id + ", was not found in Nebula's database. This likely means your mod was never uploaded to Nebula.\nYou need to upload at least one version to Nebula (public or private) in order to manage mod members.\nIt can also be caused by a network error." , "Mod ID not in Nebula", MessageBox.MessageBoxButtons.OK);